openapi-cli/src/main/java/org/ballerinalang/ballerina/OpenApiConverterUtils.java

```diff
@@ -293,7 +293,7 @@ private static String checkAvailabilityOfGivenName(String openApiName, File[] li
         for (File file : listFiles) {
             if (file.getName().equals(openApiName)) {
                 String userInput = System.console().readLine("There is already a/an " + file.getName() +
-                        " in the location. Do you want to override the file [Y/N]? ");
+                        " in the location. Do you want to override the file [N]? ");
```

Comment:
Why have we removed `[Y]es` option in here?

Reply:
Capitalize represents the default/recommended option. I removed this sort of prompts due to avoiding user confusion in between default value[Yes and No]. Could you please suggest some suitable way to represent these two values? fixed for this issue [138](https://github.com/ballerina-platform/ballerina-openapi/issues/138)

Reply:
Yeah I agree. But capitalizing the default option does not meaning removing the alternative IMO. How about using `Do you want to override the file? (y/N) `?

Reply:
Yeah, this way looks good to me. User can explicitly get the default value here. I will change with this.